lib/ret/project.ex

```diff
@@ -110,21 +110,21 @@ defmodule Ret.Project do
   def changeset(
         %Project{} = project,
         account,
-        %OwnedFile{} = project_owned_file,
-        %OwnedFile{} = thumbnail_owned_file,
+        project_owned_file,
```

Comment:
Since this is now optional we don't want the typecheck here. It is still checked since we are now using `put_assoc` instead of `put_change`, which will check the type matches the association.